tapir/coop/templates/coop/log/create_share_ownerships_log_entry.html

```diff
@@ -3,4 +3,4 @@
     ending {{ entry.end_date }}
 {% else %}
     without end date
-{% endif %}
+{% endif %}
```

Comment:
This change seems unnecessary. You mind need to configure editor to add a new line at the end of the file (on save).

Reply:
This file comes from Frederik. I cannot figure out, why I made the change at the end. But {% endif %} is necessary at the as closing tag. Or did I understand you wrong?

Reply:
@kerstenkenan Could please check settings of PyCharm (if you use it for development of Tapir), that you have enabled "Ensure every saved file ends with a line break"? It's inside `Preferences > Editor > General`. Saving the file afterwards should add a new line and this problem will go away.

Reply:
I use VSCode . But why do I need a last empty line? Is this some kind of convention?

Reply:
Yes, apparently it is a convention that is here in Tapir. You didn't have that kind of setting in VSCode, so after saving the file, you made an unnecessary change. And we should avoid that, so people don't commit back and forth removals/additions of new lines at the end of the files.

Reply:
Done. And I also found the settings in VSCode for this.